### ISSUE TITLE
Unable to open PDF files on Chrome

### ISSUE THREAD

**Issue by Popolechien** [kiwix/kiwix-tools#683]
I'm testing recently created [Nautilus zim files](https://dev.library.kiwix.org/viewer#moldova-ministry-education_ron_clasa3_2024-06/home) on dev.library.kiwix.org and when I try to browse the corresponding PDF they get blocked by Chrome:
![image](https://github.com/kiwix/kiwix-tools/assets/21675885/d584c4e9-3c19-4c6a-a461-3b5c9383eb75)

**Comment by kelson42** [comment]
duplicate of #670?

**Comment by Popolechien** [comment]
Nope, different problem.

**Comment by rgaudin** [comment]
I can reproduce and it is not iframe-related as this happens also when browsing on `/content`.

It does not happen on [direct links](https://dev.library.kiwix.org/content/moldova-ministry-education_ron_clasa3_2024-06/files/Clasa%20III/Limba%20englez%C4%83_Manual_Clasa_III.pdf) though.

I tested on a recent Chrome (126), might be interesting to test with an older versions. I see in the inspector that it has a `Sec-Fetch-Mode: navigate` request header. Server doesn't care but it indicate how Chrome considers this.
Is it possible that Chrome now prevents _navigating_ to PDF document ?? Right-clicking open in new tab works as expected…

<img width="1446" alt="Screenshot 2024-06-13 at 08 08 59" src="https://github.com/kiwix/kiwix-tools/assets/57929/96b23dc8-08ba-4adb-935f-87ae6676df95">

**Comment by mgautierfr** [comment]
Tested with a random from `https://www.gutenberg.org/files/29801/` (clicking to `https://www.gutenberg.org/files/29801/29801-pdf.pdf`).

pdf open correctly.

Difference seems to be on `Cache-Control` and `Strict-Transport-Security` (but not sure it is actually the root cause)


![image](https://github.com/kiwix/kiwix-tools/assets/86161/84ac6f0d-56f4-444e-803d-2b4a75a99c17)

**Comment by mgautierfr** [comment]
Removing the `target="_blank" on the link in the node tree make the pdf opening correctly in the current tab...

**Comment by kelson42** [comment]
@veloman-yunkan So it is time to complete this and wanted to check/update the issue. First of all the latest version of the ZIM reported in the issue is available at https://mirror.download.kiwix.org/zim/.hidden/dev/moldova-ministry-education_ron_clasa3_2025-03.zim.

For the rest I tested with Chromium on Ubuntu (so a recent version of both Ubuntu and Chromium) and I confirm that the bug is still there:

![Image](https://github.com/user-attachments/assets/156dafda-3844-4c9b-a4b0-860dcf2a4b6b)
